docs/cave/bundles-and-procedures.md

```diff
@@ -1,48 +1,48 @@
-# Bundles and Procedures
-AWIPS contains two methods for saving and loading data resources: **Bundles** are a simple way to save loaded resources to access in future CAVE sessions.  **Procedures** are similar to Bundles, but can be thought of as *groups of bundles* and allows the user to manage saved resources with more control.
+# Displays and Procedures
+AWIPS contains two methods for saving and loading data resources: **Displays** are a simple way to save loaded resources into a bundle file to access in future CAVE sessions.  **Procedures** are similar to Displays, but can be thought of as *groups of displays* and allows the user to manage saved resources with more control.
 
 
-##  Bundles
+##  Displays
 
 ### **File > Load Display**
 
-Load a previously-saved bundle from within the AWIPS system.  The Open Bundle dialog allows you to select your own saved bundles as well as those saved by other users.
+Load a previously-saved display from within the AWIPS system.  The pop-up dialog allows you to select your own saved displays as well as those saved by other users.
 
-Each selected bundle will load its contents to new tabs which are named after the bundle file name (e.g. NAM_ThetaE).
+Each selected display will load its contents to new tabs which are named after the display file name (e.g. GFS_TempWind).
```

Comment:
Hmmm, so watchin the current gif that's on our website, this functionality has changed slightly.  In the current documentation gif, when the bundle is loaded, it creates a new tab with that name. In your new gif, it no longer does this?  So maybe just remove this sentence?  Or change it to just say the display will load onto the screen... Also, what happens if you load one bundle, and then load another?  Does it replace it entirely?  If so maybe, say something along those lines.... loading a bundle will replace everything in the current map view?  (Also, if you open a different map tab and then load the bundle, does it load into that tab?)   The functionality seems to have changed a little so I just want it to be clear exactly what loading the bundle does now...

Reply:
Hmmm yes, I see what you mean. Michael was changing this functionality of CAVE a lot....So in the NWS version there is a "Editor Display" and then a "Perspective Display". The functionality in Michael's gif isn't really either of the default options. 

"Editor Display" - it only saves the current tab (editor) and when it loads, it replaces whatever is in the current editor with was was previously saved.
 "Perspective Display" - it saves all your tabs and if you were in the 5 pane layout everything loaded in those maps and when it loads it replaces everything you have open.

Current Unidata functionality is closer to the  Perspective Display - it saves all current tabs and when it loads it replaces all existing tabs with the saved ones. It doesn't do anything with the side 4 pane. But anyways I will update the wording to use:
"When loading a display, all existing tabs will be closed and replaced with the saved contents."

Reply:
Okay gotcha.  Yeah I think as long as our documentation is clear on what's happening, it's probably fine if it does something different now than before (haven't heard any big complaints from users yet at least).